repair/row_level.cc

```diff
@@ -2928,6 +2929,7 @@ repair_service::repair_service(distributed<gms::gossiper>& gossiper,
     , _sys_dist_ks(sys_dist_ks)
     , _sys_ks(sys_ks)
     , _view_update_generator(vug)
+    , _tm(tm)
```

Comment:
Here, instead of keeping a reference to the task_manager, I think it would be better to keep a `task_manager::module`
and construct it here e.g. by `_task_manager_module(tm, "repair")` and it's constructor can use a task_manager method to register the module in the (shard) module map, and deregister the module in the destructor.

The module can be used from this point on to interface the task manager such that the service will use the module methods rather than the task_manager methods to create and control tasks.